jax_galsim/bounds.py

```diff
@@ -126,10 +128,12 @@ def includes(self, *args):
         else:
             raise TypeError("include takes at most 2 arguments (%d given)" % len(args))
 
-    def expand(self, factor):
+    def expand(self, factor, yfactor=None):
```

Comment:
any reason to have a different naming convention than the `expand` in the original Galsim? (uses `factor_x` and `factor_y`)

Reply:
The function works the same though so not very important

Reply:
fixed!